ros/src/util/packages/autoware_rviz_plugins/rosbag_controller/src/autoware_rosbag_plugin.cpp

```diff
@@ -165,7 +165,7 @@ void Autoware_Rosbag_Plugin::on_button_record_start_clicked()
 
   /* Caculate max split size */
   if (split_size != 0 && ui->checkBox_split_size->isChecked())
-    recoParam.max_size = ceil(split_size * 1048576 * 1024);
+    recoParam.max_size = ceil(split_size * 1000 * 1000 * 1000);
```

Comment:
You have changed from gibibytes to gigabytes. Please explain why.

Reply:
Because Ubuntu directly show data size in gigabytes. If we use gibibytes, it will lead to inconsistent  of our input and recorded data size showed by Ubuntu, like we set split size as 20GB but obtain a 21.5GB data. 
Some user report this issue to us, and say they are confused by this. So we decide to change to  gigabytes.

Reply:
Your PR description does not mention that at all. Please update the description to state what you have actually changed and why.

Reply:
My bad. Thank you for your reminding. I update the PR now.